src/almanack/cli.py

```diff
@@ -149,7 +149,9 @@ def check(
         if almanack_score_metrics["almanack-score"] != 1:
 
             # introduce a table of output in CLI
-            print("The following Software Gardening Almanack metrics have failed:")
+            print(
+                "The following Software Gardening Almanack metrics may be helpful to address:"
```

Comment:
```suggestion
                "The following Software Gardening Almanack metrics may be helpful to remedy:"
```

Reply:
```suggestion
                "The following Software Gardening Almanack metrics may be helpful to improve upon your repository:"
```
Maybe good to add something about quality or longevity. I think something along these lines, where we are looking to only "enhance" or "improve upon" their repository, is a softer blow to someone. We want a user to feel like the work they have done is still good, but more can be done so they can keep their hardwork alive for longer.

Reply:
Thanks - I like the idea of "improve" here and will apply that change.